src/libratbag-data.c

```diff
@@ -53,6 +53,12 @@ enum driver {
 	STEELSERIES,
 };
 
+enum ratbag_device_type {
+	MOUSE = 0,
+	KEYBOARD,
+	TYPE_NONE
```

Comment:
```c
enum ratbag_device_type {
	NONE = 0,
	MOUSE,
	KEYBOARD,
}
```

Reply:
I get a compiler error because driver also has `NONE` 

![Screenshot_20200228_185422](https://user-images.githubusercontent.com/16562333/75572796-c90fc880-5a5b-11ea-9750-37a9afe57748.png)

Reply:
What about renaming it to UNKOWN?

Reply:
hmm yeah, or maybe `INVALID` ?

Reply:
Also possible, but I think `INVALID` implies an error, which it doesn't really do.